src/files/ThemeXml.ts

```diff
@@ -0,0 +1,149 @@
+import { Archive } from '../classes/Archive.ts';
+import { XmlFile } from '../classes/XmlFile.ts';
+import { FileMime } from '../enums.ts';
+import { QNS } from '../utilities/namespaces.ts';
+import { evaluateXPathToMap } from '../utilities/xquery.ts';
+import { create } from '../utilities/dom.ts';
+
+/**
+ * Represents the various 'scheme' elements that comprise a theme.
+ *
+ * Currently, only FontScheme is implemented. It functions as a fallback for
+ * determining which fonts to apply when the StylesXml cannot determine which
+ * font to apply for the required Normal style in MS Word.
+ *
+ * @TODO Implement ColorScheme and FormatScheme
+ */
+export type ThemeElements = {
+	fontScheme: FontScheme;
+}
+
+export type FontScheme = {
+	name: string;
+	majorFont: {
+		latinFont: LatinFont;
+		otherFonts: Font[];
+	};
+	minorFont: {
+		latinFont: LatinFont;
+		otherFonts: Font[]
+	}
+}
+
+export type Font = {
+	script?: string;
+	typeface: string;
+}
+
+export interface LatinFont extends Font {
+	panose: string;
+}
+
+export class ThemeXml extends XmlFile {
+	public static contentType = FileMime.theme;
+	public readonly themeElements: ThemeElements;
```

Comment:
Possibly you can record `fontScheme` (and in the future `colorScheme` and `formatScheme` directly on the class? `themeElements` itself seems to have no purpose aside from containing that information. It would make the object structure a lot flatter!

Reply:
@wvbe I had considered this, too. Right now, what you're saying makes sense to do (and so I'll probably go ahead and make this change). But if we want the class to follow the structure of the XML inside `themes1.xml`, there are other elements (not implemented here/yet) that exist as siblings of `<a:themeElements/>`.

To me, it felt like a six-of-one, half-dozen of another choice. But for now, I agree 👍